transmart-rest-api/src/test/groovy/org/transmartproject/rest/logging/ChildProcessAppenderTests.groovy

```diff
@@ -101,6 +102,7 @@ class ChildProcessAppenderTests extends Specification {
         thrown ChildFailedException
     }
 
+    @Ignore
```

Comment:
Can you also document here in the code, why the test is being ignored temporarily?